lib/internal/bootstrap/pre_execution.js

```diff
@@ -100,6 +100,8 @@ function patchProcessObject(expandArgv1) {
   addReadOnlyProcessAlias('_preload_modules', '--require');
   addReadOnlyProcessAlias('noDeprecation', '--no-deprecation');
   addReadOnlyProcessAlias('noProcessWarnings', '--no-warnings');
+  addReadOnlyProcessAlias('noExperimentalWarnings',
+                          '--no-experimental-warnings');
```

Comment:
If this happens, please don’t use this. This is only for legacy flags that already have corresponding entries on the `process` object.

Reply:
Thanks, good to know. I'll look into the code more for the correct place to add the flag.

Reply:
@codeman869 You can look around for `getOptionValue()` in our source tree, that should help :)